### PR TITLE
ccvm: remove legacy support

### PR DESCRIPTION
**Pull request by rbradford**
Remove support for loading legacy VM specification and metadata.

Fixes: #18

Signed-off-by: Rob Bradford <robert.bradford@intel.com>

### CODE REVIEW

**Comment by rbradford** [comment]
@markdryan is this what you had in mind for this issue?

File: ccvm/workload.go [hunk 1]
```diff
@@ -189,9 +189,6 @@ func restoreWorkload(ws *workspace) (*workload, error) {
 	var wkld workload
 	data, err := ioutil.ReadFile(path.Join(ws.instanceDir, "state.yaml"))
 	if err != nil {
-		if err = wkld.spec.VM.loadLegacyInstance(ws); err != nil {
-			return nil, err
-		}
 		return &wkld, nil
```
Comment (markdryan) [inline ccvm/workload.go]: This should be simply return nil, err

Comment (rbradford) [reply ccvm/workload.go]: Yeh, oops, fixed.

**Comment by markdryan** [comment]
There's still one reference to ciao-down that needs to be removed

https://github.com/intel/ccloudvm/blob/master/ccvm/prepare.go#L205

**Pull request merged**